src/main/resources/application.properties

```diff
@@ -29,3 +29,5 @@ app.env.egenAnsattV1Endpoint=${VIRKSOMHET_EGENANSATT_V1_URL}
 # From config map "loginservice-idporten"
 app.env.loginserviceIdportenAudience=${LOGINSERVICE_IDPORTEN_AUDIENCE}
 app.env.loginserviceIdportenDiscoveryUrl=${LOGINSERVICE_IDPORTEN_DISCOVERY_URL}
+app.env.naisAadDiscoveryUrl=${AZURE_APP_WELL_KNOWN_URL:null}
```

Comment:
burde man ha deafult valu null her?
Føler det er ryddigere og crashe på startup

Reply:
Har ingen sterke meninger her. Ser det er satt som default null i **veilarboppfolging**, så her må nesten dere ta en avgjørelse på hva dere ønsker.

Reply:
når jeg gjør det i veilarbaktivitet får jeg

java.lang.IllegalStateException: Failed to load ApplicationContext
...
Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'aapen-brukernotifikasjon-nyBeskjed-v1' in value "${aapen-brukernotifikasjon-nyBeskjed-v1}"